ci-operator/config/openshift/openshift-tests-private/README.md

```diff
@@ -50,6 +50,7 @@ Job frequency is defined by cron according to the test requirements
 - f4: every 4 days
 - f5: every 5 days
 - ...
+- f999: disable the job temporarely
```

Comment:
```suggestion
- f999: disable the job temporarily
```

Reply:
Good catch. Fix in https://github.com/openshift/release/pull/58217